pages/solid-events.md

```diff
@@ -6,16 +6,25 @@ permalink: /events
 
 # Solid Events
 
-Solid Events are an opportunity for people working on Solid or who are interested about learning more about Solid to meet in person locally. Solid Events are run by people like you. If you are interested in coordinating a Solid Event near you reach out to the Solid Manager on info@solidproject.org.
+Solid Events are an opportunity for people who are working on Solid or are interested in Solid to meet. Solid Events are run by people like you. If you are interested in coordinating a Solid Event near you, reach out to the Solid Manager at [info@solidproject.org](mailto:info@solidproject.org).
 
-In light of the COVID-19 pandemic Solid Events are moving online for the foreseeable future. Tune in on the first Thursday of every month at 1600 CET to join.
 
-Are you interested in speaking about what you have been building on Solid? Reach out to the Solid Manager on [info@solidproject.org](mailto:info@solidproject.org) to get a speaking slot at the next event.
+{:.has-background-light .has-text-danger}
+> In light of the COVID-19 pandemic, Solid Events are moving online for the foreseeable future. Join Solid World on the first Thursday of every month at 1600 CET.
```

Comment:
Cool, I didn't know you could apply classes like that in Jekyll/Liquid.

That said, the colour contrast doesn't meet the WCAG standards for accessible text, making it unreadable for many people, so I'd suggest to use the following component that's built-in to Bulma and does have sufficient contrast:

```suggestion
<div class="message is-info">
  <p class="message-body">
    In light of the COVID-19 pandemic, Solid Events are moving online for the foreseeable future. Join Solid World on the first Thursday of every month at 1600 CET.
  </p>
</div>
```